lib/configurators/aws/aws.go

```diff
@@ -1118,6 +1127,13 @@ func isTargetAWSAssumeRoleForDatabases(databases []*servicecfg.Database, targetA
 	})
 }
 
+func isTargetAWSAssumeRoleForResourceMatchers(resourceMatchers []services.ResourceMatcher, targetARN string) bool {
+	for _, resourceMatcher := range resourceMatchers {
+		return resourceMatcher.AWS.AssumeRoleARN == targetARN
```

Comment:
this returns whether the first one matches

Reply:
good catch!